lib/utils/which.rb

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+
+# Cross-platform way of finding an executable in the $PATH.
+#
+#   which('ruby') #=> /usr/bin/ruby
+def which(cmd)
```

Comment:
I don't think we want this implemented as a global method.  If this were in a module, say `Inspec::Util::Which` which provided a method `which` (as defined below) it would be cleaner.

Then, in each resource (or even in the resource superclass) we could do `include Inspec::Util::Which` without defining it everywhere on all objects. Surely some object somewhere implements an unrelated `which`, and it would be bad to potentially clobber it.